docker-compose.yml

```diff
@@ -11,8 +11,10 @@ services:
             - python
             - python-sync
             - redis
+            - cherami
         environment:
-            - WAIT_FOR=go,node,python,python-sync
+            - WAIT_FOR=go,node,python,python-sync,cherami
```

Comment:
thanks @breerly. Now I changed to health check based dependency in docker-compose.yaml and it's working fine. Next step for me: try to reduce the startup time for cassandra/cherami